Tests/test_GenBank_unittest.py

```diff
@@ -432,6 +433,43 @@ def test_genbank_date_invalid(self):
             gb = SeqIO.read(handle, "gb")
             self.assertEqual(gb.annotations["date"], "01-JAN-1980")
 
+    def test_longer_locus_line(self):
+        """Check that we can read and write files with longer locus lines"""
+        # Create file
+        with open(path.join("GenBank", "DS830848.gb"), 'r') as inhandle:
+            data = inhandle.readlines()
+            data[0] = "LOCUS       AZZZAA02123456789 10000000000 bp    DNA     linear   PRI 15-OCT-2018\n"
+        with open(path.join("GenBank", "long_header_genbank_test.gb"), 'w') as outfile:
+            outfile.writelines(data)
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("error", BiopythonParserWarning)
+            try:
+                record = SeqIO.read(path.join("GenBank", "long_header_genbank_test.gb"), 'genbank')
+            except BiopythonParserWarning as e:
+                self.assertEqual(str(e), "Attempting to parse locus line that is extra long or malformed  :\n"
+                                 "'LOCUS       AZZZAA02123456789 10000000000 bp    DNA     linear   PRI 15-OCT-2018\\n'\n"
+                                 "Found locus 'AZZZAA02123456789' size '10000000000' residue_type 'DNA'\n"
+                                 "Some fields may be wrong.")
+            else:
+                self.assertTrue(False, "Expected specified BiopythonParserWarning here.")
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            record = SeqIO.read(path.join("GenBank", "long_header_genbank_test.gb"), 'genbank')
+            SeqIO.write(record, path.join("GenBank", "long_header_genbank_test_out.gb"), 'genbank')
+
+            record_in = SeqIO.read(path.join("GenBank", "long_header_genbank_test_out.gb"), 'genbank')
+            self.assertEqual(record_in.id, "DS830848.1")
+            self.assertEqual(record_in.name, "AZZZAA02123456789")
+            self.assertEqual(len(record_in.seq), 10000000000)
+
+        # Clean up
```

Comment:
These never get called if the test fails, this kind of thing would be better in the ``tearDown`` method (perhaps with the ``setUp`` method used to setup a temp folder or filenames). Or even better, never create these files in the first place.